content/docs/command-reference/plots/diff.md

```diff
@@ -6,76 +6,79 @@ plotting them in a single image.
 ## Synopsis
 
 ```usage
-usage: dvc plots diff [-h] [-q | -v] [-t [TEMPLATE]] [-d [DATAFILE]] [-f FILE]
-                     [-s SELECT] [-x X] [-y Y] [--stdout] [--no-csv-header]
-                     [--no-html] [--title TITLE] [--xlab XLAB] [--ylab YLAB]
-                     [revisions [revisions ...]]
+usage: dvc plots diff [-h] [-q | -v] [--targets [<path> [<path> ...]]]
+                      [-t <name_or_path>] [-x <field>] [-y <field>]
+                      [--no-csv-header] [--title <text>]
+                      [--x-label <text>] [--y-label <text>] [-o <path>]
+                      [--show-vega]
+                      [revisions [revisions ...]]
 
 positional arguments:
-  revisions             Git commits to plot from
+  revisions             Git commits to plot from/to
 ```
 
 ## Description
 
-This command visualize difference between metrics among experiments in the
-repository history. Requires that Git is being used to version the metrics
-files.
+This command is a way to visualize the difference between metrics among
```

Comment:
Let's write a sentence or two that it's not a "difference" - it's just multiple graphs on the same plot?

Reply:
Sure. If it's not a difference though, what's the point of having 2 commands for showing plots? This one could be renamed show and show deprecated, no? WDYT @efiop

Reply:
Noted that diff is not a real diff in 6bcf06da for now.

> p.s. this is another note I remember having written before... May be lost in some other PR and will have to be careful to merge them correctly... or maybe a bad merge somewhere un-did some work here 😢

Reply:
p.s. found a parallel ongoing discussion related to this in https://github.com/iterative/dvc/pull/3963#discussion_r436649649

Reply:
Extracted as an extra bullet in #1414